test/telemetry.test.ts

```
@@ -190,4 +190,22 @@
       expect.any(Number),
     );
   });
+
+  it('should accept webui_action as a valid event type', () => {
+    delete process.env.PROMPTFOO_DISABLE_TELEMETRY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)